marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.tsx

```diff
@@ -37,11 +37,20 @@ const CreateEditMappingDialog = (props) => {
       setSrcQuery(props.mapData.sourceQuery);
       setSelectedSource(props.mapData.selectedSource);
       if(props.mapData.selectedSource === 'collection'){
-      let srcCollection = props.mapData.sourceQuery.substring(
-          props.mapData.sourceQuery.lastIndexOf("[") + 2, 
-          props.mapData.sourceQuery.lastIndexOf("]") - 1
-      );
-      setCollections(srcCollection);
+      if(props.mapData.sourceQuery.includes('[') && props.mapData.sourceQuery.includes(']')) {
```

Comment:
I think the real issue here is that sourceQuery is expecting a list when it should expect a singular value.
The input field was a list in QuickStart, taking multiple collections. For HubCentral I think we decided to go with 1 collection, so not a list.
@rjrudin  @xnikhil08 please correct me if I am wrong.